specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Language - Conversational Language Understanding Authoring APIs (2022-05-01)",
+    "version": "2022-05-01",
+    "description": "The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms. The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/overview</a>."
+  },
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "description": "A subscription key for a Language service resource.",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/language",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "common.json#/parameters/Endpoint"
+      }
+    ]
+  },
+  "paths": {
+    "/authoring/analyze-conversations/projects": {
```

Comment:
Is the /authoring segment necessary? Looking at questionanswering-authoring.json from 2021-05-01-preview, it establishes a pattern where you'd just use /analyze-conversations/projects instead. This won't conflict with your inference api /:analyze-conversations since it's an operation endpoint.

Reply:
Initially we didn't have the "/authoring/" prefix, but this caused conflicts in our application gateway (APIM) where the different language configurations would conflict with each other.

E.g. The authoring would use /language/analyze-text/projects as a prefix. The inference would use /language as a prefix and have an operation with path analyze-text/jobs. This operation would conflict with the authoring's prefix and return 404. We have a thread talking about this and a discussion was done a while back. @johanste was part of the discussion and attended the meeting that reached the "/language/authoring" as a prefix conclusion.